Core/Core/Views/HorizontalMenuViewCell.swift

```diff
@@ -41,6 +41,7 @@ class HorizontalMenuViewCell: UICollectionViewCell {
 
     override var isSelected: Bool {
         didSet {
+            textLabel.accessibilityLabel = isSelected ? "selected \(textLabel.text ?? "")" : textLabel.text
```

Comment:
Can we instead use the [selected](https://developer.apple.com/documentation/uikit/uiaccessibility/uiaccessibilitytraits/1620197-selected) accessibility trait?

Reply:
Great suggestion.  I hadn't used that one much.  Thanks for bringing it to my attention.